index.html

```diff
@@ -1,74 +1,174 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
-
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Fun Bus Travel Agency - Solution</title>
 
   <link href="https://fonts.googleapis.com/css?family=Indie+Flower|Roboto" rel="stylesheet">
   <link href="css/index.css" rel="stylesheet">
-
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
 
 <body>
 
-  
-     
+  <!-- Start of Navigation -->
+  <nav>
+    <div class="navigation">
+      <div class="logo">
+        <h1>Fun Bus</h1>
+      </div>
+      <ul class="nav-links">
+        <li class="nav-item"><a href="#">Home</a></li>
+        <li class="nav-item"><a href="#">About Us</a></li>
+        <li class="nav-item"><a href="#">Blog</a></li>
+        <li class="nav-item"><a href="#">Contact</a></li>
+      </ul>
+    </div>
+  </nav>
+  <!-- End of Navigation -->
+
+  <!-- Start of Home Container -->
   <div class="container home">
-   
+
+    <!-- Start of Header -->
+    <header class="hero">
+      <img src="img/fun-bus.jpg" alt="Picture of toy bus">
+      <h2>Welcome To Fun Bus!</h2>
+      <p> Lorem ipsum dolor sit, amet consectetur adipisicing elit. Culpa ad assumenda vel ducimus cumque maxime
+        perspiciatis repellat voluptas blanditiis nam ut adipisci voluptatibus, deleniti aspernatur dolorem debitis
+        sapiente ipsum hic.</p>
+    </header>
+    <!-- End of Header -->
+
+    <!-- Start of Content Section -->
     <section class="content-section">
+
+      <!-- Content Piece 1 -->
       <div class="text-content">
         <h2>Let's Go!</h2>
-        <p>Adventure webdesign pretty design design, excursion cute WordPress blogger design webdesign adventure. Pretty simple traveling fun WordPress wanderlust darn simple organized.</p> 
-        <p>Expedition colorful design simple excursion blogger blogger design WordPress design, design organized website theme.</p>
+        <p>Adventure webdesign pretty design design, excursion cute WordPress blogger design webdesign adventure. Pretty
+          simple traveling fun WordPress wanderlust darn simple organized.</p>
+        <p>Expedition colorful design simple excursion blogger blogger design WordPress design, design organized website
+          theme.</p>
       </div>
       <div class="img-content">
         <img src="img/adventure.jpg" alt="Let's go on an adventure!">
       </div>
     </section>
+
+    <!-- Content Piece 2 -->
     <section class="content-section inverse-content">
       <div class="img-content">
-          <img src="img/fun.jpg" class="img-fluid rounded" alt="Lets have fun!">
+        <img src="img/fun.jpg" class="img-fluid rounded" alt="Lets have fun!">
```

Comment:
When writing alt tags - try to describe what's in the image. It's used for accessibility purposes. If the browser has to read it out loud it won't be able to describe the picture here